pkg/sql/util/index_util.go

```diff
@@ -37,6 +37,8 @@ func BuildIndexTableName(unique bool, indexNumber int, indexName string) string
 	}
 	name += strconv.Itoa(indexNumber)
 	name += "_"
+	name += tblName
```

Comment:
Building index table name with tblName seems to be a bad idea -- what if people do alter table rename?   Can table id work?

Reply:
If I first create table t1, then obtain TableId through t1, and then construct index table, then t1's definition of index is wrong. Do you have any good suggestions

Reply:
If I first create table t1, then obtain TableId through t1, and then construct index table, then t1's definition of index is wrong. Do you have any good suggestions

Reply:
done